api/src/main/java/com/github/streamshub/console/api/service/MetricsService.java

```diff
@@ -114,6 +118,12 @@ public PrometheusAPI createClient(ConsoleConfig consoleConfig, KafkaClusterConfi
         return null;
     }
 
+    Optional<TlsConfiguration> getTlsConfiguration(String sourceName) {
+        String dotSeparatedSource = "metrics.source." + replaceNonAlphanumeric(sourceName, '.');
+        String dashSeparatedSource = "metrics-source-" + replaceNonAlphanumeric(sourceName, '-');
```

Comment:
The name may be slightly different (dots versus dashes) depending on whether the property was set via a system property or an environment variable.